test/e2e/framework/networking_utils.go

```diff
@@ -218,8 +218,9 @@ func (config *NetworkingTestConfig) DialFromContainer(protocol, containerIP, tar
 // pod and confirm it doesn't show up as an endpoint.
 func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targetPort, maxTries, minTries int, expectedEps sets.String) {
 	var cmd string
+	// Make the command timeout after 1s (no matter what protocol it is).
 	if protocol == "udp" {
-		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 3 nc -w 1 -u %s %d", targetIP, targetPort)
+		cmd = fmt.Sprintf("echo 'hostName' | timeout -t 1 nc -w 1 -u %s %d", targetIP, targetPort)
```

Comment:
@bprashanth - it seems that even with -t here it doesn't work. The failures:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/35114/pull-kubernetes-e2e-gce/1100/

are definitely related.

Do you know why?